Topic/uh agent section (#12)
OSSystems/updatehub-docs: docs/updatehub-agent/intro.md

> The **UpdateHub Agent** is the application that works on the device and is the responsible for update proccess, checking for updates packages, connecting to the server (Cloud or other), installing the packages and rebooting the device.

Reviewer: ```suggestion
The **UpdateHub Agent** runs on the device and is responsible for managing the whole update process.
```

> In addition to providing the activities mentioned above, the **Updatehub Agent** makes available to the user a series of settings that allow flexibility in the update processes of your device.

Reviewer: ```suggestion
In addition to that, the **Updatehub Agent** makes available to the user several settings that allow flexibility in the update processes of your device.
```
